src/index.html

```diff
@@ -5,10 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="one">
+        <img
+          src = "images/moyo_1.svg"
+          alt = "logo"
+          class = "logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav___list">
+          <li>
```

Comment:
It should be nav__item too, and in the css you just need to remove margin for first child, use pseudo selectors